docs/live-migration.md

```diff
@@ -0,0 +1,235 @@
+# Live Migration
+
+## Introduction
+
+The Live Migration feature allows [kubevirt](kubevirt.io) virtual machines to be [live migrated](https://kubevirt.io/user-guide/operations/live_migration/)
+while keeping the established TCP connections alive, and preserving the VM IP configuration.
+These two requirements provide seamless live-migration of a KubeVirt VM using OVN-Kubernetes
+cluster default network.
+
+## Requirements
+
+- KubeVirt >= v1.0.0
+- DHCP aware guest image (fedora family is well tested, https://quay.io/organization/containerdisks)
+
+# Limitations
+
+- Only KubeVirt VMs with bridge binding pod network are supported
+- Single stack IPv6 is not supported
+- DualSack do not configure routes for IPv6 over DHCP/autoconf
```

Comment:
```suggestion
- DualSack does not configure routes for IPv6 over DHCP/autoconf
```

Reply:
further question: this means the routes must manually configured routes ?

Reply:
Yes, the router advertisment support at ovn-k will advertise LRP but we need the arp proxy default gateway wich is not advertised, but since the default gw is stable across nodes we can configure it at general ignition at hypershift or cloud-init at other envs.